tests/Unit/Console/Commands/ValidateConfigCommand/HandleTest.php

```diff
@@ -67,6 +67,13 @@ public function command_can_be_run_with_the_files_option(): void
     /** @test */
     public function error_is_displayed_if_a_config_validation_fails(): void
     {
+        Config::set('cache.default', ['invalid', 'items', 'in', 'an', 'array']);
```

Comment:
I noticed that some of the tests are failing, and I think it is because the framework itself is trying to use this value (and crashing) when it initializes the Cache Manager, so the test isn't actually being run.